dagster_university/dagster_and_dbt/dagster_and_dbt_tests/conftest.py

```diff
@@ -0,0 +1,29 @@
+import os
+import subprocess
+
+import pytest
+
+
+def _dbt_command(cmd: str, lesson: str):
+    dir = os.path.join(os.path.dirname(__file__), f"../dagster_and_dbt/{lesson}/analytics")
+
+    cmd = ["dbt", cmd, "--project-dir", dir, "--profiles-dir", dir]
+    print(cmd)
+    result = subprocess.run(cmd, capture_output=True, text=True)
+    print(result.stdout)
+
+    if result.returncode != 0:
+        pytest.fail(f"dbt command failed: {result.returncode}")
+
+
+@pytest.fixture(autouse=True)
+def no_requests(monkeypatch):
+    monkeypatch.setenv("DUCKDB_DATABASE", "data/staging/data.duckdb")
+
+
+def pytest_configure(config):
```

Comment:
Really hate this work around but for the tests to work, the dbt manifest needs to exist. I was unable to get this to work with fixtures so needed to pull it out to the configuration level. So it is slightly annoying that running the tests will execute `dbt deps` and `dbt parse` for ALL lessons before the tests are run. But works for now.

Reply:
Can you add this explanation as a comment in the code itself?